lib/ui/components/cv_text_field.dart

```diff
@@ -5,11 +5,14 @@ class CVTextField extends StatelessWidget {
   final String label;
   final TextEditingController controller;
   final TextInputType type;
+  final TextInputAction action;
   final int maxLines;
   final String initialValue;
   final Function(String) validator;
   final Function(String) onSaved;
+  final Function(String) fieldSubmitted;
```

Comment:
Follow the naming conventions for functional callbacks, `fieldSubmitted` -> `onFieldSubmitted`.